src/inverseRobot.js

```diff
@@ -7,7 +7,15 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  for (const key in robot) {
+    const value = robot[key];
+    if (inverse[value] !== undefined) {
```

Comment:
This would be more suitable here
```suggestion
    if (inverse.hasOwnProperty(value)) {
```